Sources/MongoSwift/ChangeStreamOptions.swift

```diff
@@ -0,0 +1,101 @@
+/// Describes the modes for configuring the fullDocument field of a
+/// `ChangeStreamDocument`.
+public enum FullDocument: RawRepresentable, Codable {
+    /// The change stream document will include both a delta describing the
```

Comment:
I think I mentioned this in the design doc, but this comment should be updated to only describe what this option is doing to the updateLookup field and can omit the parts about the `UpdateDescription` field.

Reply:
@patrickfreed I don't think I'm fully understanding the issue here. By option do you mean the `ChangeStreamOptions.fullDocument`, not the `FullDocument` type? And are you referring to the docstring for the `updateLookup` case?